tests/command/commands/team_test.py

```diff
@@ -466,3 +467,105 @@ def test_handle_edit_lookup_error(self):
                                                           "edit brs", user),
                                   (self.testcommand.lookup_error, 200))
         self.db.store.assert_not_called()
+
+    def test_handle_refresh_not_admin(self):
+        """Test team command refresh parser with insufficient permission."""
+        test_user = User(user)
+        self.db.retrieve.return_value = test_user
+        with self.app.app_context():
+            self.assertTupleEqual(self.testcommand.handle("team refresh",
+                                                          user),
+                                  (self.testcommand.permission_error, 200))
+        self.db.store.assert_not_called()
+
+    def test_handle_refresh_lookup_error(self):
+        """Test team command refresh parser with lookup error."""
+        test_user = User(user)
+        team = Team("BRS", "brs", "brS")
+        test_user.permissions_level = Permissions.admin
+        self.db.retrieve.return_value = None
+        self.db.retrieve.side_effect = LookupError
+        self.gh.org_get_teams.return_value = [team]
+        with self.app.app_context():
+            self.assertTupleEqual(self.testcommand.handle("team refresh",
+                                                          user),
+                                  (self.testcommand.lookup_error, 200))
+        self.db.store.assert_not_called()
+
+    def test_handle_refresh_github_error(self):
+        """Test team command refresh parser with github error."""
+        test_user = User(user)
+        test_user.permissions_level = Permissions.admin
+        self.db.retrieve.return_value = test_user
+        self.gh.org_get_teams.side_effect = GithubAPIException("error")
+        with self.app.app_context():
+            self.assertTupleEqual(self.testcommand.handle("team refresh",
+                                                          user),
+                                  ("Refresh teams was unsuccessful with "
+                                   "the following error: error", 200))
+        self.db.store.assert_not_called()
+
+    def test_handle_refresh_changed(self):
+        """Test team command refresh parser if team edited in github."""
+        test_user = User(user)
+        test_user.permissions_level = Permissions.admin
+        team = Team("TeamID", "TeamName", "android")
+        team_update = Team("TeamID", "new team name", "android")
+        team2 = Team("OTEAM", "other team2", "ios")
+
+        self.db.retrieve.return_value = test_user
+        self.db.query.return_value = [team, team2]
+
+        # In this case, github has an updated version of team!
+        remote_team_update = mock.Mock(GithubTeam.Team)
+        remote_team_update.id = team_update.github_team_id
+        remote_team_update.name = team_update.github_team_name
+
```

Comment:
One could be more stringent: 
```suggestion
remote_member1 = mock.Mock(github.NamedUser.NamedUser)
remote_member1.setFields(member1)
remote_team_update.members = PaginatedList(remote_member1)
```
here, but it feels unnecessary